test/provisioner/provision.go

```diff
@@ -64,6 +77,113 @@ type InstallOverlay interface {
 	Edit(ctx context.Context, cfg *envconf.Config, properties map[string]string) error
 }
 
+// Waiting timeout for bringing up the pod
+const PodWaitTimeout = time.Second * 15
```

Comment:
I think we need more than 15 seconds like 30 seconds or something because for me the provisioning failed on this.

```bash
✗ ./caa-provisioner-cli -action=provision
INFO[0000] Creating VPC...
...
INFO[0139] Install Kbs
Wait pod 'kbs-659c77bf67-xfd84' status for Ready
ERRO[0155] All pods are not running: timed out waiting for the condition
FATA[0155] All pods are not running: timed out waiting for the condition
```